gateway-server/src/test/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateServiceTest.java

```diff
@@ -195,7 +204,8 @@ public void testAddAndRemoveTokenIncludesCache() throws Exception {
    * Verify that the token state reaper includes token state which has not been cached, so it's not left in the keystore
    * forever.
    */
-  @Test
+  @Ignore("I'm not sure if this is a valid use case since we have everything in the cache when eviction takes place")
```

Comment:
@pzampino As you can see I only ignored this test case for now because I'm not sure if it makes sense going forward. Since the eviction thread checks everything in the cache we may remove this test case. What do you think?